js/app/src/main.ts

```diff
@@ -16,7 +20,24 @@ let FONTS: string | [];
 
 FONTS = "__FONTS_CSS__";
 
+//@ts-ignore
+let IndexComponent;
 function create_custom_element(): void {
+	const o = {
+		SvelteComponent: svelte.SvelteComponent
+	};
+	for (const key in svelte) {
+		if (key === "SvelteComponent") continue;
+		if (key === "SvelteComponentDev") {
+			//@ts-ignore
+			o[key] = o["SvelteComponent"];
+		} else {
+			//@ts-ignore
+			o[key] = svelte[key];
+		}
+	}
+	//@ts-ignore
+	window.__gradio__svelte__internal = o;
```

Comment:
Pull into function with descriptive name.

Need to add defs for these window objects.